cli/server/server.go

```diff
@@ -606,15 +609,16 @@ Main:
 // In case global Address (of the node) provided and RPC/Prometheus/Pprof don't have configured addresses they will
 // use global one. So Node and RPC and Prometheus and Pprof will run on one address.
 func configureAddresses(cfg *config.ApplicationConfiguration) {
-	if cfg.Address != "" {
+	// TODO: use Addresses instead.
```

Comment:
Or keep it just for compatibility, `Address/Port` only. If there are any `Addresses` used, don't do this at all, otherwise we'll have ambiguous configurations like this:
```
P2P:
  Addresses:
    - X.Y.Z.1:20333
RPC:
  Addresses:
    - :20332
```
Is it `0.0.0.0:20332` or `X.Y.Z.1:20332`? The intention might be any of these.

Reply:
Fixed, current behaviour is derive IP only for deprecated BasicServeces fields in case if deprecated `Address` is set.